cmd/fossa/cmd/report/attribution.go

```diff
@@ -0,0 +1,107 @@
+package report
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/olekukonko/tablewriter"
+	"github.com/urfave/cli"
+
+	"github.com/fossas/fossa-cli/api/fossa"
+	"github.com/fossas/fossa-cli/cmd/fossa/display"
+	"github.com/fossas/fossa-cli/cmd/fossa/flags"
+	"github.com/fossas/fossa-cli/cmd/fossa/setup"
+	"github.com/fossas/fossa-cli/config"
+	"github.com/fossas/fossa-cli/errors"
+)
+
+var attributionCmd = cli.Command{
+	Name:  "attribution",
+	Usage: "Generate attribution report",
+	Flags: flags.WithGlobalFlags(flags.WithAPIFlags(flags.WithOptions(flags.WithReportTemplateFlags([]cli.Flag{
+		cli.BoolFlag{Name: JSON, Usage: "format output as JSON"},
+	})))),
+	Action: generateReport,
+}
+
+func generateReport(ctx *cli.Context) error {
+	err := setup.SetContext(ctx, true)
+	if err != nil {
+		log.Fatalf("Could not initialize: %s", err.Error())
+	}
+	defer display.ClearProgress()
+
+	display.InProgress(fmt.Sprint("Fetching Attribution Report"))
+
+	locator := fossa.Locator{
+		Fetcher:  config.Fetcher(),
+		Project:  config.Project(),
+		Revision: config.Revision(),
+	}
+	attrReport, err := fossa.GetReport(locator)
+	if err != nil {
+		return errors.Wrapf(err, "Unable to build report for for project %s:", locator)
+	}
+
+	fmt.Fprintf(os.Stdout, `Generated by fossa-cli (https://github.com/fossas/fossa-cli).
+Attribution for project %s, revision %s
```

Comment:
This is a little difficult to read and would look better if we broke the formatted string out as a constant similar to how we've done with the templates in the other report files. maybe something like 
```
const attributionTemplate = `Generated by fossa-cli (https://github.com/fossas/fossa-cli).
Attribution for project %s, revision %s
Direct Dependencies
%s
Deep Dependencies
%s`
```